lib/get-config.js

```diff
@@ -22,8 +22,22 @@ const ALLOWED_ERROR_CODES = [
 ];
 
 function requirePlugin(pluginName, fromConfigPath) {
-  // throws exception if not found
-  let pluginPath = resolve.sync(pluginName, { basedir: path.dirname(fromConfigPath) });
+  let pluginPath;
+
+  function getPluginPath(name) {
+    return resolve.sync(name, { basedir: path.dirname(fromConfigPath) });
+  }
+
+  try {
+    // throws exception if not found
+    pluginPath = getPluginPath(pluginName);
+  } catch (error) {
+    try {
+      pluginPath = getPluginPath(`ember-template-lint-plugin-${pluginName}`);
+    } catch (_) {
+      throw error;
+    }
+  }
```

Comment:
I think we'd need to structure this a bit different. The cases I'd like to ensure are handled are:

* we only attempt to resolve with the fuller name for `MODULE_NOT_FOUND` types of errors, not any possible error
* when called with `foo`, if neither `foo` nor `ember-template-lint-plugin-foo` are found the resulting error should mention `foo` (or should mention that we checked for both and found neither)

Reply:
Indeed, the logic was not quite right. It should be good now :)

I also added a test case to make sure `foo` > `ember-template-lint-plugin-foo`